### ISSUE TITLE
[Feature Request] testing support for Edge

### ISSUE THREAD

**Issue by johnml1135** [plotly/dash#1946]
Currenlty it appears that only Chrome and Firefox are supported for testing:

https://github.com/plotly/dash/blob/6ee3eeecf302498418bca2c026ba42af8e2dd5fb/dash/testing/plugin.py#L24

Some corporate environments lock down Chrome at the preference of Edge.  having full support for testing with Edge in pytest would be very helpful.

**Comment by alexcjohnson** [comment]
Good idea! Looks relatively straightforward https://www.geeksforgeeks.org/automated-browser-testing-with-edge-and-selenium-in-python/ and https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/

Feel like making a PR to add this? 🙏
